chunky/src/java/se/llbit/chunky/main/Chunky.java

```diff
@@ -353,28 +343,28 @@ public boolean sceneInitialized() {
   }
 
   @PluginApi
-  public void setRendererFactory(RendererFactory rendererFactory) {
-    this.rendererFactory = rendererFactory;
+  public void setRendererFactory(RenderManagerFactory renderManagerFactory) {
```

Comment:
Should this be deprecated now? I don't know about other plugins but for me the only reason I needed this was to [replace the render manager with something that can switch the underlying renderer](https://github.com/chunky-dev/chunky-denoiser/blob/master/src/main/java/de/lemaik/chunky/denoiser/DenoiserPlugin.java#L37).